cli/aws_orbit/models/context.py

```diff
@@ -172,6 +172,7 @@ class TeamContext:
 
     # Manifest default
     authentication_groups: Optional[List[str]] = None
+    profiles: Optional[List[Dict[str, Union[str, Dict[str, Any]]]]] = None
```

Comment:
we leave this for backwards compatibility? did you try removing it? i think we have our SerDe configured to ignore extra parameters so it _should_ ignore any existing Profiles attributes

Reply:
Needed for backward compatibility when I was flopping back-and-forth between versions...intended to remove it in 1.4 release